derp/derp_server.go

```diff
@@ -1398,3 +1416,13 @@ func writePublicKey(bw *bufio.Writer, key *key.Public) error {
 	}
 	return nil
 }
+
+func (s *Server) Stat() {
```

Comment:
I don't think you'll need this. All the state can be kept in local variables of the long-running /stat ServeHTTP call.